.gitignore

```diff
@@ -21,3 +21,8 @@ cscope*
 .vscode
 
 *.log
+
+color-map-gpi
+memory/
+old_results/
+plots/
```

Comment:
Please remove items that are specific to your paper (not part of the library). If you want you can leave a ignored folder `results` that serves as a catch all folder for when doing evaluations in the library.